libs/sdk-backend-base/src/decoratedBackend/index.ts

```diff
@@ -174,6 +197,11 @@ export type ExecutionDecoratorFactory = (executionFactory: IExecutionFactory) =>
  */
 export type CatalogDecoratorFactory = (catalog: IWorkspaceCatalogFactory) => IWorkspaceCatalogFactory;
 
+/**
+ * @alpha
+ */
+export type SecuritySettingsDecoratedFactory = (securitySettings: ISecuritySettingsService) => ISecuritySettingsService;
```

Comment:
typo: decorator factory (because the factory creates decorators for existing services :))

Reply:
Fixed, my bad.